e2e/src/step_definitions/media-viewer.ts

```diff
@@ -111,6 +111,12 @@ When('I select a text on pdf doc', async () => {
 });
 
 Then('I expect text highlight popup should appear', async function () {
```

Comment:
this is not actually checking if the highlight popup appears

Reply:
Changed the phrase to 'I capture the text highlight popup' - as at this point , I want to see the popup appears and capture this log.